.ci/build_kubernetes.sh

```diff
@@ -9,18 +9,25 @@ echo ${DOCKER_PASSWORD} | docker login --username ${DOCKER_USERNAME} --password-
 FIRST_DOMAIN_HOSTNAME=${JOB_NAME}.${DEVELOPMENT_SERVER_DOMAIN}
 SECOND_DOMAIN_HOSTNAME=2.${JOB_NAME}.${DEVELOPMENT_SERVER_DOMAIN}
 
-# Set domain name into ingress controller so ingress can listen on domain name
-yq write --inplace project-base/kubernetes/ingress.yml spec.rules[0].host ${FIRST_DOMAIN_HOSTNAME}
-yq write --inplace project-base/kubernetes/ingress.yml spec.rules[1].host ${SECOND_DOMAIN_HOSTNAME}
-
-# Set domain into dev-stack (redis-admin, elasticsearch, adminer) hostnames
-yq write --inplace project-base/kubernetes/kustomize/overlays/ci/ingress-patch.yaml [0].value.host adminer.${FIRST_DOMAIN_HOSTNAME}
-yq write --inplace project-base/kubernetes/kustomize/overlays/ci/ingress-patch.yaml [1].value.host elasticsearch.${FIRST_DOMAIN_HOSTNAME}
-yq write --inplace project-base/kubernetes/kustomize/overlays/ci/ingress-patch.yaml [2].value.host redis-admin.${FIRST_DOMAIN_HOSTNAME}
+FILES=(
+    project-base/kubernetes/ingress.yml
+    project-base/kubernetes/kustomize/overlays/ci/ingress-patch.yaml
+    project-base/kubernetes/deployments/webserver-php-fpm.yml
+)
+VARS=(
+    FIRST_DOMAIN_HOSTNAME
+    SECOND_DOMAIN_HOSTNAME
+)
 
-# Set domain into webserver hostnames
-yq write --inplace project-base/kubernetes/deployments/webserver-php-fpm.yml spec.template.spec.hostAliases[0].hostnames[+] ${FIRST_DOMAIN_HOSTNAME}
-yq write --inplace project-base/kubernetes/deployments/webserver-php-fpm.yml spec.template.spec.hostAliases[0].hostnames[+] ${SECOND_DOMAIN_HOSTNAME}
+for FILE in "${FILES[@]}"
```

Comment:
It would make more sense to me to first copy and modify the files in `project-base/app/config/`, build and push the images and then replace all the variables in Kubernetes manifests and Kustomize config in one loop.

Reply:
So you want to push sensitive information like database password to external docker repository? Probably you do not need to set all parameters before push, but you can do that after push to docker repository.

Reply:
@henzigo I understand your concern, but it's already done like that, I'm just speaking about making it simpler without changing the resulting behavior. The parameters don't actually contain sensitive information as the Kubernetes infrastructure doesn't allow access from outside network.

But generally speaking, it would be best to have info in `parameters.yml` not pushed to the Docker repository (ideally not saved in a file at all as per https://12factor.net/config).

Reply:
@PetrHeinz We are pushing only default parameters from `parameters.yml.dist` and then we are overriding them with our custom parameters.

You are right that Kubernetes infrastructure doesn't allow access from outside network, but we are using services like `adminer` where you can use those parameters.